news/2023/2023-01-19-gmt-apps-are-now-open

```diff
@@ -0,0 +1,39 @@
+
+![](/wiki/shared/news/banners/gmt-generic.jpg)
+
+It's been over 2 years since the last GMT application cycle, and while some things changed around, one thing remains the same - we may be looking for you!
+
+The Global Moderation Team is reponsible for keeping the community in a good condition, with a place for every single being. The GMT is covering a handful of niches, most notable ones being in-game chat and forum moderation, let's not ignore the other parts though. The greennamed peers are also there to help moderate osu! subcommunities, such as beatmapping one. The members of the team will also give you assistance with some of your technical issues. The inherent goal for us is to reach closer and closer to the living community.
+
+That being said, this time we will look not only for chat or forum moderators. If you feel you're a prominent figure in the osu! community and would like to help us with the community moderation side of things, be sure to shoot us an application as well!
+
+While ususally we seek new members in a [process of recomendations and nominations](https://osu.ppy.sh/wiki/en/People/The_Team/Global_Moderation_Team#becoming-a-moderator), we're also opening apps from time to time to look for fresh blood we might be not noticing - not everyone is so outspoken and perceptible at the end of the day! While we will still continue using the our classic methods, we believe that such diversification of lookup methods is vastly valueable to the team's health.
+
+Before applying, though, be sure to take a look of a list of factors regarding your application below:
+- **You must be at least 18 years old. This is NOT negotiable.** The GMT role means that you will have to deal with content of inappropriate nature that's not suitable for all ages.
+- **Your account must be at least one year old.** This is to ensure you are somewhat familliar with the osu! community at very least.
+- You must be active within osu! community. While we will not stick to chat or forums metrics explcitly (at least when it's not your everyday area of focus), we expect you to be active and engaged in certain communities.
+- If selected, you will need to adhere to [Contributor Covenant Code of Conduct](https://osu.ppy.sh/wiki/en/Rules/Contributor_Code_of_Conduct).
+
+While we will look into every single application, and we count on you to apply, there are certain areas that require more moderation manpower in particular. We're actively seeking moderators for the following channels: **#arabic, #bulgarian, #czech, #greek, #hebrew, #italian, #romanian, #turkish**.
+
+Think we are waiting for you? **[Click here to apply!](https://form-auth.ppy.sh/form/2023-gmt-apps)** After an osu! OAuth verification, you will be able to fill a Google Forms questionnaire, be sure to reserve some time as you will surely need some time to answer the questions we prepared for you!
```

Comment:
also the last sentence feels a bit jumbled with "be sure to ... as you will surely ..."
```suggestion
Think we are waiting for you? **[Click here to apply!](https://form-auth.ppy.sh/form/2023-gmt-apps)** After authentication, you will be able to fill a Google Forms questionnaire, be sure to reserve some time as you will surely need some time to answer the questions we prepared for you!
```